lib/controller/StudyRequestController.js

```diff
@@ -139,23 +137,23 @@ StudyRequestController.push({
   method: 'PUT',
   path: '/requests/study/{id}',
   options: {
+    auth: {
+      scope: [AuthScope.STUDY_REQUESTS_EDIT.name],
+    },
     response: {
       schema: StudyRequest.read,
     },
     validate: {
       params: {
         id: Joi.number().integer().positive().required(),
       },
-      payload: StudyRequest.update.append({
-        // TODO: remove when we have RBAC
-        isSupervisor: Joi.boolean().default(false),
-      }),
+      payload: StudyRequest.update,
```

Comment:
this is the only use of `StudyRequest.update` - but in this case, we need the distinction with `StudyRequest.read`, as a study request being updated might contain both new studies and previously existing studies.

longer-term, we should probably standardize this pattern across our models.  one possible approach: define a `Model` class that includes one or more `ModelColumn` objects.  a `ModelColumn` is:

- a flag that determines whether this is a transient or persisted column;
- one of:
  - a Joi schema used to validate this column;
  - another `Model` (X-to-one relationships);
  - an array containing a `Model` as its only element (X-to-many relationships).

note that this does not manage any SQL / CRUD stuff - this is *only* about validation, though we could add a lightweight CRUD layer.

anyways - definitely out of scope here, but some food for thought.